core-stack-api/templates/configmap-env.yaml

```diff
@@ -4,10 +4,7 @@ kind: ConfigMap
 metadata:
   name: {{ include "orchestrate-api.fullname" . }}-envmap
   labels:
-    app.kubernetes.io/name: {{ include "orchestrate-api.name" . }}
```

Comment:
Labels are important to identify assets across namespaces or scopes(service,pods,ingresses..). This one `app.kubernetes.io/name` sounds like the standard way to identify them. DId you validate this label id was not use somewhere else to fetch this config maps?

Reply:
They are still present through the injection of `{{- include "orchestrate-api.labels" . | nindent 4 }}` and 

```
{{- define "orchestrate-api.labels" -}}
helm.sh/chart: {{ include "orchestrate-api.chart" . }}
{{ include "orchestrate-api.selectorLabels" . }}
{{- if .Chart.AppVersion }}
app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
{{- end }}
app.kubernetes.io/managed-by: {{ .Release.Service }}
{{- end }}

{{/*
Selector labels
*/}}
{{- define "orchestrate-api.selectorLabels" -}}
app.kubernetes.io/name: {{ include "orchestrate-api.name" . }}
app.kubernetes.io/instance: {{ .Release.Name }}
{{- end }}
```